recipes/lensfunpy/meta.yaml

```diff
@@ -18,17 +19,25 @@ build:
 requirements:
   build:
     - {{ compiler('cxx') }}
-    - lensfun
-    
+    - lensfun <0.3.95
+        
   host:
     - pip
     - python
-    - lensfun
+    - lensfun <0.3.95
     - numpy
     
   run:
     - python
-    
+    - cython
+    - nose
+    - wheel >=0.25
+    - sphinx
+    - sphinx_rtd_theme
+    - scipy
+    - lensfun ==0.3.2
```

Comment:
Why the exact pin here and not above?

Reply:
copied the run deps part from Olivia's meta, where as the others were already there from mine. Do you have a preference as to which to use?